boot/boot_serial/src/boot_serial.c

```diff
@@ -24,7 +24,6 @@
 
 #include "sysflash/sysflash.h"
 
-#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_INFO
```

Comment:
I don't agree with this change.

MCUboot is close to its flash size on Zephyr targets, and being able to control log levels on a per-translation-unit basis is the best tool I know of to manage this, as extra logging calls carry nontrivial flash penalties.

Reply:
Well, one can always add those lines back and have the previous functionality. It's just not enforced in `HEAD` now.